lib/dialects/postgres/query.js

```diff
@@ -95,8 +95,8 @@ class Query extends AbstractQuery {
         return queryResult;
```

Comment:
A few changes

1) Please run tests against v7
2) Remove note about v7 incompatibility from https://github.com/sequelize/sequelize/blob/master/README.md#installation and https://github.com/sequelize/sequelize/blob/master/docs/usage.md#postgresql